app/controllers/barcode_printers_controller.rb

```diff
@@ -72,38 +72,38 @@ def destroy
   end
   # This module define common behavior used by other controller to print things
   module Print
-    def print_asset_labels(succes_url, failure_url)
-      assets = params[:printables]
-      prefix = nil
-      unless assets.nil?
-        printables = []
-        assets = assets.keys
-        assets.sort{ |a,b| b.to_i <=> a.to_i }.each do |id|
-          asset = Asset.find(id)
-          prefix = asset.prefix
-          unless asset.barcode.present?
-          asset.barcode = AssetBarcode.new_barcode
-          asset.save!
-          end
-          printables.push PrintBarcode::Label.new({ :number => asset.barcode, :study => asset.name_for_label.to_s, :prefix => prefix, :suffix => "" })
-         end
-
-         unless printables.empty?
-           BarcodePrinter.print(printables, params[:printer], prefix)
-        end
-      end
-      flash[:notice] = "Your labels have been sent to printer #{params[:printer]}."
-      redirect_to succes_url
-
-    rescue Savon::Error, Sanger::Barcode::Printing::BarcodeException => e
-      if e.kind_of? Savon::Error
-        flash[:warning] = "There is a problem with the selected printer. Please report it to Systems."
-      else
-        flash[:error] = "There was a problem with the printer. Select another and try again."
-      end
-      Rails.logger.error($!)
-
-      redirect_to failure_url
+  #   def print_asset_labels(succes_url, failure_url)
```

Comment:
Probably best to just delete code, rather then commenting it out, and remove it entirely. The old code still exists in Git, so can be retrieved if we need to restore it. You can still comment stuff out when actively working on it, just try and remove it before committing.